src/components/ViewIndividualArticle.jsx

```diff
@@ -0,0 +1,116 @@
+import { useEffect, useState } from 'react';
+import { getArticles, getUser } from '../utils/api';
+import styles from '../styles/ViewIndividualArticle.module.css';
+import formatDate from '../utils/ArticleCard.utils';
+import {
+  TfiTag,
+  TfiComment,
+  TfiAngleDown,
+  TfiAngleUp,
+  TfiTrash,
+} from 'react-icons/tfi';
+
+const ViewIndividualArticle = ({ article_id }) => {
+  const [isLoading, setIsLoading] = useState(true);
+  const [individualArticle, setIndividualArticle] = useState({});
+  const [date, setDate] = useState('');
+  const [userAvatarImg, setUserAvatarImg] = useState('');
```

Comment:
Your state held on line 16 here are derivable - as in you can access them directly from the article object - I would remove them and just use `article.suchAndSuch` when you need - otherwise, state updates cause the component to re-render unnecessarily